### ISSUE TITLE
warnings generated from npm install

### ISSUE THREAD

**Issue by michaelpalumbo** [worldmaking/node-gles3#3]
Hi @grrrwaaa 

I get the following in the console, and the install script cannot finish:

```shell
> gles3@0.0.1 preinstall /Users/michaelpalumbo/node-gles3
> node tools/gen.js

unhandled GLFWgammaramp* ramp   void glfwSetGammaRamp(GLFWmonitor* monitor, GLFWgammaramp* ramp)
unhandled GLFWimage* images     void glfwSetWindowIcon(GLFWwindow* window, int count, GLFWimage* images)
unhandled double timeout        void glfwWaitEventsTimeout(double timeout)
unhandled double* xpos  void glfwGetCursorPos(GLFWwindow* window, double* xpos, double* ypos)
unhandled double* ypos  void glfwGetCursorPos(GLFWwindow* window, double* xpos, double* ypos)
unhandled double xpos   void glfwSetCursorPos(GLFWwindow* window, double xpos, double ypos)
unhandled double ypos   void glfwSetCursorPos(GLFWwindow* window, double xpos, double ypos)
unhandled GLFWdropfun cbfun     GLFWdropfun glfwSetDropCallback(GLFWwindow* window, GLFWdropfun cbfun)
unhandled double time   void glfwSetTime(double time)

> gles3@0.0.1 install /Users/michaelpalumbo/node-gles3
> node-gyp rebuild

  CXX(target) Release/obj.target/gles3/src/node-gles3.o
warning: unknown warning option '-Wno-unused-but-set-variable'; did you mean
      '-Wno-unused-const-variable'? [-Wunknown-warning-option]
1 warning generated.
  SOLINK_MODULE(target) Release/gles3.node
  CXX(target) Release/obj.target/glfw3/src/node-glfw3.o
warning: unknown warning option '-Wno-unused-but-set-variable'; did you mean
      '-Wno-unused-const-variable'? [-Wunknown-warning-option]
In file included from ../src/node-glfw3.cpp:3:
../src/node-glfw3.h:1318:20: warning: variable 'cursor' is uninitialized when used
      here [-Wuninitialized]
        glfwDestroyCursor(cursor);
                          ^~~~~~
../src/node-glfw3.h:1316:20: note: initialize the variable 'cursor' to silence
      this warning
        GLFWcursor* cursor;
                          ^
                           = nullptr
../src/node-glfw3.h:1334:24: warning: variable 'cursor' is uninitialized when used
      here [-Wuninitialized]
        glfwSetCursor(window, cursor);
                              ^~~~~~
../src/node-glfw3.h:1332:20: note: initialize the variable 'cursor' to silence
      this warning
        GLFWcursor* cursor;
                          ^
                           = nullptr
../src/node-glfw3.cpp:368:24: warning: variable 'timeout' is uninitialized when
      used here [-Wuninitialized]
        glfwWaitEventsTimeout(timeout);
                              ^~~~~~~
../src/node-glfw3.cpp:366:16: note: initialize the variable 'timeout' to silence
      this warning
        double timeout;
                      ^
                       = 0.0
../src/node-glfw3.cpp:463:27: warning: variable 'xpos' is uninitialized when used
      here [-Wuninitialized]
        glfwGetCursorPos(window, xpos, ypos);
                                 ^~~~
../src/node-glfw3.cpp:460:14: note: initialize the variable 'xpos' to silence this
      warning
        double* xpos;
                    ^
                     = nullptr
../src/node-glfw3.cpp:463:33: warning: variable 'ypos' is uninitialized when used
      here [-Wuninitialized]
        glfwGetCursorPos(window, xpos, ypos);
                                       ^~~~
../src/node-glfw3.cpp:461:14: note: initialize the variable 'ypos' to silence this
      warning
        double* ypos;
                    ^
                     = nullptr
../src/node-glfw3.cpp:480:27: warning: variable 'xpos' is uninitialized when used
      here [-Wuninitialized]
        glfwSetCursorPos(window, xpos, ypos);
                                 ^~~~
../src/node-glfw3.cpp:477:13: note: initialize the variable 'xpos' to silence this
      warning
        double xpos;
                   ^
                    = 0.0
../src/node-glfw3.cpp:480:33: warning: variable 'ypos' is uninitialized when used
      here [-Wuninitialized]
        glfwSetCursorPos(window, xpos, ypos);
                                       ^~~~
../src/node-glfw3.cpp:478:13: note: initialize the variable 'ypos' to silence this
      warning
        double ypos;
                   ^
                    = 0.0
../src/node-glfw3.cpp:497:1: warning: control may reach end of non-void function
      [-Wreturn-type]
}
^
../src/node-glfw3.cpp:496:51: warning: variable 'cbfun' is uninitialized when used
      here [-Wuninitialized]
        GLFWdropfun result = glfwSetDropCallback(window, cbfun);
                                                         ^~~~~
../src/node-glfw3.cpp:494:19: note: initialize the variable 'cbfun' to silence
      this warning
        GLFWdropfun cbfun;
                         ^
                          = nullptr
../src/node-glfw3.cpp:543:14: warning: variable 'time' is uninitialized when used
      here [-Wuninitialized]
        glfwSetTime(time);
                    ^~~~
../src/node-glfw3.cpp:541:13: note: initialize the variable 'time' to silence this
      warning
        double time;
                   ^
                    = 0.0
11 warnings generated.
  SOLINK_MODULE(target) Release/glfw3.node
  CXX(target) Release/obj.target/openvr/src/node-openvr.o
warning: unknown warning option '-Wno-unused-but-set-variable'; did you mean
      '-Wno-unused-const-variable'? [-Wunknown-warning-option]
1 warning generated.
  SOLINK_MODULE(target) Release/openvr.node

> gles3@0.0.1 postinstall /Users/michaelpalumbo/node-gles3
> node renderToFBO.js

glfw 3.3.0
glfw version-string: 3.3.0 Cocoa NSGL
```

**Comment by grrrwaaa** [comment]
CLosing -- this is the same as #2.

**Comment by grrrwaaa** [comment]
Oops misread, no it isn't, but the fix to #2 should also solve this.

**Comment by michaelpalumbo** [comment]
ok
